Jenkinsfile_CNP

```diff
@@ -7,7 +7,7 @@ def product = "rpts"
 def component = "frontend"
 
 def yarnBuilder = new uk.gov.hmcts.contino.YarnBuilder(this)
-def branchesToSync = ['demo', 'perftest', 'ithc']
+def branchesToSync = ['demo']
```

Comment:
you will need it to sync otherwise these changes in this PR won't get added to those branches and then it won't remove the infra in the environments we are removing